### PR TITLE
Imagestream tags

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1818790

CP for 4.2, 4.3, 4.4, 4.5

Changed sections:

https://imagestream-tags--ocpdocs.netlify.app/openshift-enterprise/latest/migration/migrating_3_4/migrating-application-workloads-3-to-4.html#migration-prerequisites_migrating-3-4

https://imagestream-tags--ocpdocs.netlify.app/openshift-enterprise/latest/migration/migrating_4_1_4/migrating-application-workloads-4_1-to-4.html#migration-prerequisites_migrating-4_1-4_x

https://imagestream-tags--ocpdocs.netlify.app/openshift-enterprise/latest/migration/migrating_4_2_4/migrating-application-workloads-4_2-to-4.html#migration-prerequisites_migrating-4_2-4_x

### CODE REVIEW

**Comment by XinRedhat** [comment]
LGTM

File: migration/migrating_4_1_4/migrating-applications-with-cam.adoc [hunk 1]
```diff
@@ -6,7 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/migration-launching-cam.adoc[leveloffset=+1]
+You can migrate application workloads by adding your clusters and replication repository to the CAM web console. Then you can create and run a migration plan.
```
Comment (ahardin-rh) [inline migration/migrating_4_1_4/migrating-applications-with-cam.adoc:9]: Then, you can

File: migration/migrating_4_2_4/migrating-applications-with-cam.adoc [hunk 2]
```diff
@@ -6,7 +6,11 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/migration-launching-cam.adoc[leveloffset=+1]
+You can migrate application workloads by adding your clusters and replication repository to the CAM web console. Then you can create and run a migration plan.
```
Comment (ahardin-rh) [inline migration/migrating_4_2_4/migrating-applications-with-cam.adoc:9]: Then, you can

File: modules/migration-creating-ca-bundle.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// migration/migrating_3_4/migrating-applications-with-cam.adoc
+// migration/migrating_4_1_4/migrating-applications-with-cam.adoc
+// migration/migrating_4_2_4/migrating-applications-with-cam.adoc
+[id='creating-ca-bundle_{context}']
+= Creating a CA certificate bundle file
+
+If you use a self-signed certificate to secure a cluster or a replication repository, certificate verification may fail with the following error message: `Certificate signed by unknown authority`.
```
Comment (ahardin-rh) [inline modules/migration-creating-ca-bundle.adoc:8]: s/may/might

**Comment by ahardin-rh** [comment]
Just some minor nits from me. Otherwise LGTM!

**Pull request merged**